tests/README.md

```diff
@@ -0,0 +1,25 @@
+# Debug and Run Tests Using Go
+
+  1. Download your level state file from azure blob storage.
+  2. Rename the file to terraform.tfstate and place it in a known location.
+  3. Create a .env file in the tests folder with the following structure.
+
+      ```env
+      STATE_FILE_PATH=<path to state file>
+      ENVIRONMENT=<deployed caf environment name>
+      ARM_SUBSCRIPTION_ID=<subscription id>
+      ```
+
+  4. Run the following command in the ./tests folder to download go dependencies.
+
+      ```go
+      go get
+      go mod tidy
+      ```
+
+  5. Run your test using go test cmd
```

Comment:
It might be a good idea to add a note that mentions that you can also debug using VS Code and the go extension.